kubernetes/client.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	osapps_v1 "github.com/openshift/api/apps/v1"
 	osproject_v1 "github.com/openshift/api/project/v1"
+	osroutesv1 "github.com/openshift/api/route/v1"
```

Comment:
Last week, we just introduced consistent patterns to our import aliases. This doesn't match the new pattern all our import alias are following now. Imports of the form .../api/ABC/vN results in import aliases of abc_vn - if it is an openshift specific import, prefix with "os"... so in this case, change this import alias to "osroutes_v1".

Reply:
we should set this in the https://github.com/kiali/kiali/blob/master/STYLE_GUIDE.adoc